mods/ts/weapons/healweapons.yaml

```diff
@@ -1,25 +1,25 @@
-Heal:
+^HealWeapon:
 	ReloadDelay: 80
 	Range: 2c849
 	Report: healer1.aud
-	ValidTargets: Infantry
-	Projectile: Bullet
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
```

Comment:
I think this would be a good time to implement a `TargetActor` warhead that only deals damage to `Target.Actor`s.  I believe this is how many of the weapons in the original game worked? In any case, it makes a lot more sense here than overriding the spread to not spread.